cpp/lib/src/MBox.cc

```diff
@@ -28,11 +29,22 @@ MBox::Message &MBox::Message::swap(Message &other_message) {
     other_message.original_host_.swap(original_host_);
     other_message.sender_.swap(sender_);
     other_message.subject_.swap(subject_);
+    std::swap(other_message.priority_, priority_);
     other_message.message_body_.swap(message_body_);
     return *this;
 }
 
 
+std::string MBox::Message::toString() const {
+    return   "reception_time: " + TimeUtil::TimeTToString(reception_time_) + "\n"
```

Comment:
multiple spaces after return statement

Reply:
Intentional in order to line things up vertically.